### PR TITLE
Add helm registry plugin

### PR DESCRIPTION
**Pull request by joejulian**
This plugin allows the use of the quay.io application registry (or a
private registry) for helm charts.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -31,6 +31,9 @@ ENV     K8S_HELM_VERSION_LATEST=$K8S_HELM_VERSION_1_6
 ENV     GOPATH /go
 ENV     GO15VENDOREXPERIMENT 1
 
+ENV     HELM_HOME=/etc/helm
```
Comment (coffeepac) [inline Dockerfile]: I am nervous about how this will interact with `k2cli tool helm`.  is it required?

Comment (venezia) [inline Dockerfile]: I think it fixed some issues running inside of jenkins @joejulian would know more

Comment (joejulian) [inline Dockerfile]: Well, it's a lot smarter than using `$HOME/.helm`. That's different between jenkins and everything else as jenkins sets $HOME to `/home/jenkins` while k2cli uses `/root`. k2cli doesn't bind mount ~/.helm to the container so all this should do is make that consistent.

Comment (joejulian) [inline Dockerfile]: Oh, yeah... I also straced helm and even with that env set, it still looks in ~/.helm first.

Comment (coffeepac) [inline Dockerfile]: okay.  onward then.

**Pull request merged**